internal/config/config.go

```diff
@@ -36,6 +42,14 @@ type HCloudClientConfiguration struct {
 	Debug    bool
 }
 
+type RobotClientConfiguration struct {
```

Comment:
```suggestion
type RobotConfiguration struct {
```

Or move the enabled flag out of the struct ?